lerna.json

```diff
@@ -7,5 +7,5 @@
       "message": "[Version Bump]: publish %s\nlerna.json\nCHANGELOG.md\n**/CHANGELOG.md"
     }
   },
-  "version": "0.14.2"
+  "version": "independent"
```

Comment:
This is the key line. Without this [change](https://github.com/lerna/lerna#independent-mode), all packages would be bumped to 1.x.

Reply:
Awesome !